tests/python_client/base/client_v2_base.py

```diff
@@ -0,0 +1,867 @@
+import sys
+from typing import Optional
+from pymilvus import MilvusClient
+
+sys.path.append("..")
+from check.func_check import ResponseChecker
+from utils.api_request import api_request
+from utils.wrapper import trace
+from utils.util_log import test_log as log
+from common import common_func as cf
+from base.client_base import Base
+
+TIMEOUT = 120
+INDEX_NAME = ""
+
+
+class TestMilvusClientV2Base(Base):
+
+    # milvus_client = None
+    active_trace = False
+
+    def init_async_milvus_client(self):
+        uri = cf.param_info.param_uri or f"http://{cf.param_info.param_host}:{cf.param_info.param_port}"
+        kwargs = {
+            "uri": uri,
+            "user": cf.param_info.param_user,
+            "password": cf.param_info.param_password,
+            "token": cf.param_info.param_token,
+        }
+        self.async_milvus_client_wrap.init_async_client(**kwargs)
+
+    def _client(self, active_trace=False):
+        """ return MilvusClient instance if connected successfully, otherwise return None"""
+        if self.skip_connection:
+            return None
+        if cf.param_info.param_uri:
+            uri = cf.param_info.param_uri
+        else:
+            uri = "http://" + cf.param_info.param_host + ":" + str(cf.param_info.param_port)
+        res, is_succ = self.init_milvus_client(uri=uri, token=cf.param_info.param_token, active_trace=active_trace)
+        if is_succ:
+            # self.milvus_client = res
+            log.info(f"server version: {res.get_server_version()}")
+        return res
+
+    def init_milvus_client(self, uri, user="", password="", db_name="", token="", timeout=None,
+                           check_task=None, check_items=None, active_trace=False, **kwargs):
+        self.active_trace = active_trace
+        func_name = sys._getframe().f_code.co_name
+        res, is_succ = api_request([MilvusClient, uri, user, password, db_name, token, timeout], **kwargs)
+        # self.milvus_client = res if is_succ else None
+        check_result = ResponseChecker(res, func_name, check_task, check_items, is_succ,
+                                       uri=uri, user=user, password=password, db_name=db_name, token=token,
+                                       timeout=timeout, **kwargs).run()
+        return res, check_result
+
+    @trace()
+    def close(self, client, check_task=None, check_items=None):
+        func_name = sys._getframe().f_code.co_name
+        res, is_succ = api_request([client.close])
+        check_result = ResponseChecker(res, func_name, check_task, check_items, is_succ).run()
+        return res, check_result
+
+    @trace()
+    def create_schema(self, client, timeout=None, check_task=None,
```

Comment:
Should we remove this unnecessary parameter `client` and use self.client directly? The same as other func

Reply:
I think it is easier for for RBAC cases if we have the client parameter. We can switch the different clients with different permissions, instead of init the clients from time to time.